src/locale/i18n.ts

```diff
@@ -91,6 +92,10 @@ export async function dynamicActivate(locale: AppLanguage) {
       await import('@formatjs/intl-pluralrules/locale-data/pt')
       break
     }
+    case AppLanguage.ru: {
+      i18n.loadAndActivate({locale, messages: messagesRu})
+      break
```

Comment:
Looking at the other language codes, I think there might be a line missing here?

`await import('@formatjs/intl-pluralrules/locale-data/ru')`

Reply:
This is due to the new changes, will apply the edits now, thanks!

Reply:
Hmm, I made these changes in my local code repository, but it still did not allow me to create a translation in the ru folder